theatre/studio/src/utils/stringify.ts

```diff
@@ -0,0 +1,23 @@
+import {tightJsonStringify} from './tightJsonStringify'
+
+export function stringify(input: any, indentJSON: boolean = true): string {
+  try {
+    return typeof input === 'string'
+      ? input
+      : typeof input === 'function'
+      ? input.toString()
+      : indentJSON
+      ? tightJsonStringify(input)
+      : JSON.stringify(input)
+  } catch (err) {
+    return input?.name || String(input)
+  }
+}
+
+export function stringifyError(err: any) {
+  return err instanceof Error
+    ? err.toString()
+    : err == null
+    ? undefined
+    : stringify(err)
+}
```

Comment:
This has a very general name. Is it intended to be THE stringify function we use? Or does it have a particular purpose? Can you make the name more specific? Why this stringify function instead of just JSON.stringify?

Please add jsdoc.

Reply:
Good questions, this, is not `JSON.stringify`, this is the "I need to create a displayable string for errors and logging of some arbitrary objects" function.